components/auth/org.wso2.carbon.auth.scim.rest.api/src/main/java/org/wso2/carbon/auth/scim/rest/api/util/ApiServiceUtils.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.wso2.carbon.auth.scim.rest.api.util;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.wso2.charon3.core.protocol.SCIMResponse;
+
+import java.util.Map;
+import javax.ws.rs.core.Response;
+
+public class ApiServiceUtils{
+
+    private ApiServiceUtils() {
+
+    }
+
+    /* Build the javax-rs response from SCIM response
+     *
+      *@param scimResponse
+     * @return Response
+     */
+    public static Response buildResponse(SCIMResponse scimResponse) {
```

Comment:
It is better to move this to abstract Service Impl or something. Having too much utility is not good Java practice.